_includes/forms/input.html

```diff
@@ -0,0 +1,18 @@
+{% assign name = include.name | default: "input" %}
```

Comment:
```suggestion
{% assign name = include.name | default: (include.label | slugify) | default: "input" %}
```

I would try slugifying the label before defaulting to "input" (which will lead to ID conflicts very easily)

Reply:
Thanks! I went with just requiring the `include.name` rather than trying to give it a default.